### ISSUE TITLE
mintstick don't detect SD Card

### ISSUE THREAD

**Issue by s-nt-s** [linuxmint/mintstick#72]
SD card is not show up on device formatter.

![imagen](https://user-images.githubusercontent.com/5998601/70074892-f33c2080-15fb-11ea-81df-683676d98f71.png)

Same issue has been reported in https://github.com/linuxmint/nemo/issues/2189

**Comment by papoteur-mga** [comment]
I think this can easyly be done by adding the connection bus "cpio" in addition to "usb".
https://github.com/linuxmint/mintstick/blob/fd46357072547fb5921f8739fb52db3b1ff84480/lib/mintstick.py#L218

**Comment by s-nt-s** [comment]
It is happening again:

![image](https://github.com/linuxmint/mintstick/assets/5998601/6580f4ce-5dd0-4b43-aafd-9380cbf9c977)

My environment:

```
$ apt version mintstick
1.5.6

$ cat /etc/issue
Linux Mint 21.2 Victoria \n \l

$ hostnamectl 
       Icon name: computer-laptop
         Chassis: laptop
Operating System: Linux Mint 21.2                 
          Kernel: Linux 5.15.0-84-generic
    Architecture: x86-64
 Hardware Vendor: Lenovo
  Hardware Model: ThinkPad X260

$ cat /etc/linuxmint/info
RELEASE=21.2
CODENAME=victoria
EDITION="Xfce"
DESCRIPTION="Linux Mint 21.2 Victoria"
DESKTOP=Gnome
TOOLKIT=GTK
NEW_FEATURES_URL=https://www.linuxmint.com/rel_victoria_xfce_whatsnew.php
RELEASE_NOTES_URL=https://www.linuxmint.com/rel_victoria_xfce.php
USER_GUIDE_URL=https://www.linuxmint.com/documentation.php
GRUB_TITLE=Linux Mint 21.2 Xfce

$ cat  /etc/os-release
NAME="Linux Mint"
VERSION="21.2 (Victoria)"
ID=linuxmint
ID_LIKE="ubuntu debian"
PRETTY_NAME="Linux Mint 21.2"
VERSION_ID="21.2"
HOME_URL="https://www.linuxmint.com/"
SUPPORT_URL="https://forums.linuxmint.com/"
BUG_REPORT_URL="http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/"
PRIVACY_POLICY_URL="https://www.linuxmint.com/"
VERSION_CODENAME=victoria
UBUNTU_CODENAME=jammy

$ cat /etc/upstream-release/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu Jammy Jellyfish"
```

**Comment by clefebvre** [comment]
I think this is fixed in the upcoming 1.6.0.

If not, run it in terminal with `mintstick -m format`. It will output a list of detected devices and their properties. This will help us identify the issue.
